cmd/bin/bin.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bin"
+	"github.com/vertgenlab/gonomics/fasta"
+	"log"
+)
+
+func binGenome(genome string, path string, binNum int, minSize int) {
+	records := fasta.Read(genome)
+	var bins map[int][]fasta.Fasta
```

Comment:
I'm trying to think through if the map data type works perfectly for bins. If I remember correctly, maps are not ordered, but that shouldn't matter here, right?

Reply:
correct, it shouldn't matter here, I'm actually numbering them as a place holder for a key anyway, so they kind of correspond to the order they are created, but in general it doesn't matter at all for this kind of function where I'm depending on the name of a contig to be access from the value anyway.